### PR TITLE
Set DEFAULT_GRUVEO_SDK_VERSION to 0.6.5

### PR DESCRIPTION
**Pull request by ruangchupa**
-  Set DEFAULT_GRUVEO_SDK_VERSION to 0.6.5
- Updated gradle settings to support Gruveo Android SDK v0.6.5
- Updated GruveoModule.java to comply new methods introduced in Gruveo v0.6.5

### CODE REVIEW

File: android/src/main/java/com/reactnativegruveo/GruveoModule.java [hunk 1]
```diff
@@ -145,6 +145,11 @@ public void recordingStateChanged(boolean us, boolean them) {
         sendEvent("recordingStateChanged", null);
     }
 
+    @Override
+    public void recordingFilename(String filename) {
+        sendEvent("recordingFilename", filename);
```
Comment (umarniz) [inline android/src/main/java/com/reactnativegruveo/GruveoModule.java]: Shouldn't `recordingFilename` also be added to: https://github.com/senseobservationsystems/react-native-gruveo/blob/master/index.d.ts#L36

and 

https://github.com/senseobservationsystems/react-native-gruveo/blob/master/index.js#L40

File: android/build.gradle [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
     jcenter()
+    maven {
+      url 'https://maven.google.com/'
+      name 'Google'
+    }
   }
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:1.1.3'
+    classpath 'com.android.tools.build:gradle:3.0.1'
```
Comment (umarniz) [inline android/build.gradle]: I see gradle has been updated and hence it requires the `maven` repo and also the `jcenter` repos.

Can you test if this is working as expected when trying to compile with NiceDay on another machine?

File: android/src/main/java/com/reactnativegruveo/GruveoModule.java (hunk 1 above)
Comment (ruangchupa) [reply android/src/main/java/com/reactnativegruveo/GruveoModule.java]: Oh yes! I missed that.

File: android/build.gradle (hunk 2 above)
Comment (ruangchupa) [reply android/build.gradle]: I'm going to check this first on another machine and will update here whether it's working or not.

File: android/src/main/java/com/reactnativegruveo/GruveoModule.java (hunk 1 above)
Comment (ruangchupa) [reply android/src/main/java/com/reactnativegruveo/GruveoModule.java]: Just pushed my update for this.

File: android/build.gradle (hunk 2 above)
Comment (ruangchupa) [reply android/build.gradle]: @coldpinguin just helped me checked on his machine and it's compiled and built.

**Comment by ruangchupa** [comment]
@umarniz I have pushed the latest changes and tested to build on another machine. Please take a look again.

**Comment by umarniz** [comment]
@ruangchupa looks good, have you tested this in our app to make sure its working fine and nothing breaks?

**Comment by ruangchupa** [comment]
Hi @umarniz sorry haven't checked back this PR for a while, I just tested this in the latest version of our app and as far as I checked everything is working fine including the group video call feature.

**Pull request merged**